kie-soup-project-datamodel/kie-soup-project-datamodel-commons/pom.xml

```diff
@@ -32,7 +32,7 @@
   <name>Project DataModel - Commons</name>
 
   <properties>
-    <osgi.Bundle-SymbolicName>org.kie.soup.project.datamodel.commons</osgi.Bundle-SymbolicName>
+    <java.module.name>org.kie.soup.project.datamodel.commons</java.module.name>
```

Comment:
Ideally one module name should _not_ be a prefix of other one to avoid split packages. Would it make sense to call the other module `org.kie.soup.project.datamodel.api`?

Reply:
Yeah, I can make that change.